### ISSUE TITLE
Docs runthrough

### ISSUE THREAD

**Issue by yhahn** [mapbox/mapbox-studio-classic#876]
We've changed quite a few things recently, do a quick sweep of source/style quickstart to make sure there's nothing rocky.

**Comment by samanpwbb** [comment]
Base updates included icon changes:
- `inspect` instead of `eye`
- `crosshair` instead of `marker` for layer centering in source mode
- `globe` instead of `home` for Map docs (wasn't documented in the fist place)

**Comment by geografa** [comment]
I'm cleaning up this source quickstart. Looks way out of date. Will tackle others this week.

**Comment by geografa** [comment]
Noticing there's no asset/img folder for mb-pages. Is that intentional? I was about to swap out images but should I continue just dropping them in Github and grabbing the URL?

**Comment by yhahn** [comment]
yep, https://github.com/mapbox/mapbox-studio/blob/mb-pages/CONTRIBUTING.md#documentation
